src/containers/VisitPage.js

```diff
@@ -360,6 +360,7 @@ class VisitPage extends Component {
             minWidth='874.5'
             style={{ padding: '14px 0' }}>
             <StepBar
+              className="testorr"
```

Comment:
Can probably remove this at this point, though I like the name. May have to steal it for myself sometime!

Reply:
Whoops!